awx/main/models/inventory.py

```diff
@@ -176,6 +167,14 @@ def get_absolute_url(self, request=None):
 
     variables_dict = VarsDictProperty('variables')
 
+    @property
+    def has_active_failures(self):
```

Comment:
Makes it not queriable :(

Reply:
`/api/v2/inventories/?hosts_with_active_failures__gt=0`